connect/cli/plugins/commerce/utils.py

```diff
@@ -0,0 +1,340 @@
+        ws.cell(n, 1, value=attachment['id'])
+        ws.cell(n, 2, value=attachment['name'])
+        _download_file(
+            client=client,
+            folder_type=attachment['folder']['type'],
+            folder_name=attachment['folder']['name'],
+            file_id=attachment['id'],
+            file_destination=os.path.join(base_path, attachment['name']),
+        )
+        progress.update(task, advance=1)
+
+
+def export_stream(
+    client,
+    stream_id,
+    active_account_id,
+    progress,
+    output_file,
+    output_path=None,
+):
+    collection = guess_if_billing_or_pricing_stream(client, stream_id)
+    if not collection:
+        console.secho(
+            f'Stream {stream_id} not found for the current account {active_account_id}.',
+            fg='red',
+        )
+        return
+
+    response = (
+        client.ns(collection)
+        .streams.filter(id=stream_id)
+        .select(
+            'context',
+            'samples',
+            'sources',
+        )
+        .first()
+    )
+
+    wb = Workbook()
+
+    output_file = validate_output_options(output_path, output_file, default_dir_name=stream_id)
+    attachments_path = os.path.join(os.path.dirname(output_file), 'attachments')
+    if not os.path.exists(attachments_path):
+        os.mkdir(attachments_path)
+
+    sample_input_path = os.path.join(os.path.dirname(output_file), 'sample', 'input')
+    if not os.path.exists(sample_input_path):
+        os.makedirs(sample_input_path)
+
+    stream_type = 'Computed' if 'sources' in response and response['sources'] else 'Simple'
+    input = response['samples'].get('input', {})
+    input_file_name = input['name'].split('/')[-1]
+    fill_general_information(
+        ws=wb.active,
+        data={
+            'Stream ID': stream_id,
+            'Stream Name': response['name'],
+            'Stream Description': response['description'],
+            'Stream Type': stream_type,
+            'Stream Category': 'Inbound'
+            if response['owner']['id'] == active_account_id
+            else 'Outbound',
+            'Computed Stream Source ID': response['sources'][0]['id']
+            if stream_type == 'Computed'
+            else '',
+            'Computed Stream Source Name': response['sources'][0]['type']
+            if stream_type == 'Computed'
+            else '',
+            'Product ID': response['context'].get('product', {}).get('id', ''),
+            'Product Name': response['context'].get('product', {}).get('name', ''),
+            'Partner ID': response['context'].get('account', {}).get('id', ''),
+            'Partner Name': response['context'].get('account', {}).get('name', ''),
+            'Marketplace ID': response['context'].get('marketplace', {}).get('id', ''),
+            'Marketplace Name': response['context'].get('marketplace', {}).get('name', ''),
+            'Pricelist ID': response['context'].get('price_list', {}).get('id', ''),
```

Comment:
This field is not filled when the computed stream is attached to a pricelist.

Reply:
In the serializer they have the pricelist, so it should work.